README.md

```diff
@@ -43,6 +45,17 @@ List of things to be done to use RubiX are:
 		fs.s3n.impl=com.qubole.rubix.hadoop1.CachingNativeS3FileSystem   
 		fs.s3.impl=com.qubole.rubix.hadoop1.CachingNativeS3FileSystem   
 
+##### Using Rubix with Spark  
+
+1. Add the Rubix jars to Driver and Executor classpaths and use the rubix implementation for hadoop.fs.s3 and hadoop.fs.s3n.  
```

Comment:
Same here. Replace version with <version>. Or `${rubix-version}`. Then the users can set an environment variable